src/utils/VerificationValidationParser.cpp

```diff
@@ -153,9 +156,62 @@ void Parser::searchFinalDefense(Result* r) {
     }
 }
 
-Result* Parser::lc(const QString* terminalOutput) {
-    return nullptr; // TODO: implement
+Result* Parser::lc(const QString* cmd, const QString* terminalOutput) {
+	Result final;
+	final.terminalOutput = *terminalOutput;
+	final.cmd = *cmd;
+	Result* ptr_final = &final;
+
+	/* Convert terminal output and cmd into string for ease of parsing */
+	string s_terminalOutput = terminalOutput->toStdString();
+	string s_cmd = cmd->toStdString();
+
+	/* Check if length of list is zero, aka does it warnings or errors*/
+	if(stoi(s_terminalOutput.substr(s_terminalOutput.find(":") + 1)) == 0) { // Contains no issues
+		final.resultCode = Result::PASSED;
+		return ptr_final;
+	}
+
+
+	string issueDescription;
+	/* Is it an error or warning? */
+	if(s_cmd.find("-d") != string::npos) { // This is a Warning
+		final.resultCode = Result::WARNING;
+		issueDescription = "Contains duplicated ID's";
+
+	}
+	else if(s_cmd.find("-m") != string::npos) { // This is an Error
+		final.resultCode = Result::FAILED; 
+		issueDescription = "Contains mismatched ID's";
+	}
+	else { // If this is neither, assuming it is unparsable
+		final.resultCode = Result::UNPARSEABLE;
+		return ptr_final;
+	}
+
+	/* Start adding the issues to list */
+	std::list<Result::ObjectIssue> temp_issues; // List for issues
+	string temp; // Variable used for storing line of entry
+	istringstream iss(s_terminalOutput);
+	getline(iss, temp); // Gets rid of length line
+	getline(iss, temp); // Gets rid of column description
+	int count;	
+	/* Parse through the results */ 
+	for(int i = 0; i < stoi(s_terminalOutput.substr(s_terminalOutput.find(":")+1)); i++) {
```

Comment:
i know compiler probably optimizes this out but ideally:
`int x = stoi(...)`
`for (int i = 0; i < x; i++) {...}`

if compiler doesn't optimize out, it'll do that all that code every loop
plus makes it easier to read and understand with good variable name (not x)